MultiLangProgGuide/MultiLangProgGuide.rst

```diff
@@ -2693,12 +2746,13 @@ Go LockS()
 
 .. code-block:: go
 
-    LockS( timeoutNsec uint64, ... *KeyT) int
+    LockS(timeoutNsec uint64, lockName ... *KeyT) int
 
-The function wraps `ydb_lock_s()`. It releases all `Locks`_ held by
+The function wraps `ydb_lock_s()`_. It releases all `Locks`_ held by
 the process, and acquires the locks whose resource names are specified
```

Comment:
Perhaps this should spawn a Programming Consideration note - the effect of LockS() releasing existing locks is process wide, not thread wide or goroutine wide but process wide.